src/oic/oic/provider.py

```diff
@@ -2027,3 +2027,24 @@ def remove_inactive_keys(self, more_then=3600):
                         kb.remove(key)
             if len(kb) == 0:
                 self.keyjar.issuer_keys[""].remove(kb)
+
+    def get_uid_by_sub(self, sub):
```

Comment:
These methods are (now) already implemented in SessionBackend, so we should use those directly.

`self.sdb.get_uid_by_sub()` whenever this method is used.

Reply:
OK